### PR TITLE
[CU_2acnz06] Fix entity token symbol string

### PR DESCRIPTION
**Pull request by amarinkovic**
### Summary

This pull request fixes the issue of improper entity token symbol derivation. 

ClickUp: [#2acnz06](https://app.clickup.com/t/2acnz06)

### CODE REVIEW

**Comment by melks** [comment]
Task linked: [CU-2acnz06 Fix entity token symbol](https://app.clickup.com/t/2acnz06)

File: contracts/EntityTokensFacet.sol [hunk 1]
```diff
@@ -113,7 +113,19 @@ contract EntityTokensFacet is EternalStorage, Controller, EntityFacetBase, IEnti
 
     function tknSymbol(address _unit) public view override returns (string memory) {
         // max len = 11 chars
-        return string(abi.encodePacked("N-", _unit.toString().substring(3), "-", address(this).toString().substring(3), "-E"));
```
[the remark this answers is not in the document]
Comment (kevin-fruitful) [reply contracts/EntityTokensFacet.sol:116]: oh, in the Strings lib, the method substring doesn't expose a param to pass in an offset. It currently just passes 0 to the internal method _substring

[the remark this answers is not in the document]
Comment (kevin-fruitful) [reply contracts/EntityTokensFacet.sol:116]: should we add a param so we can pass in an offset?

[the remark this answers is not in the document]
Comment (amarinkovic) [reply contracts/EntityTokensFacet.sol:116]: Agreed, I will add this to align with the corresponding JS function behaviour:

```solidity
    function substr(string memory _base, int256 _start, int256 _length) internal pure returns (string memory) {
        return _substring(_base, _length, _start);
    }

```

[the remark this answers is not in the document]
Comment (amarinkovic) [reply contracts/EntityTokensFacet.sol:116]: actually, I see that overloading is no issue, so we can just go with `substring(...)`

**Pull request merged**